### ISSUE TITLE
Balasana: Unlabeled button not properly read by screen readers.

### ISSUE THREAD

**Issue by arinoch** [Automattic/themes#4746]
## Steps to replicate

1: Install and activate the Balasana theme.

2: Turn on a screen reader.

3: Use its navigational commands to move about the page, starting from the top.

## Result

A button that is used to visually collapse/expand the menu in this theme is unlabeled, and screen readers are unable to properly identify it.

## Expected

That button should either 1: say what it's intended to do (for example, "collapse/expand menu") or 2: not be visible to screen readers, as it only serves to hide the menu visually.

* User report: 26439768-hc
* Follow-up ticket: 4334433-zd-woothemes
The relevant code that needs to be changed:
`<input type="checkbox" role="button" aria-haspopup="true" id="toggle" class="hide-visually">  | <label for="toggle" id="toggle-menu" class="button">
  | Menu			<span class="hide-visually expanded-text">expanded</span>
  | <span class="hide-visually collapsed-text">collapsed</span>
  | </label>`
